packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fluid-internal/devtools-browser-extension",
-	"version": "2.0.0-internal.6.0.0",
+	"version": "2.0.0-internal.5.4.0",
```

Comment:
This value shouldn't get reduced. The version on the `next` branch is `2.0.0-internal.6.0.0`, which we'll want to keep. I suspect reverting this line will fix the issue you're running into.

Reply:
I see! Let me revert it and try again. I wonder where this change came from

Reply:
5.4.0 is the current specified version in `main`. 6.0.0 in `next` represents the next release we will be doing. I think the changes to that line were merged using the value from main instead of the one from next.

Reply:
Gotcha. `npm run check:versions` passed. Thanks for guiding me.

Reply:
Happy to help :)